containers/Dockerfile.base-centos7

```diff
@@ -19,5 +21,15 @@ SHELL ["/bin/bash", "-l", "-c"]
 CMD ["bash"]
 # switch to 'easybuild' user
 USER easybuild
+# install EasyBuild as a module (& clean up /scratch)
+RUN eb --install-latest-eb-release --prefix /scratch --installpath /easybuild && rm -rf /scratch/*
 # print info on Lmod & EasyBuild
-RUN ml --version && which -a eb && eb --version & eb --show-system-info
+RUN ml use /easybuild/modules/all && ml EasyBuild && ml --version && which -a eb && eb --version & eb --show-system-info
+# remove globally installed EasyBuild, we can use the module going forward
+USER root
+RUN pip3 uninstall -y easybuild
+# switch to 'easybuild' user
+USER easybuild
+WORKDIR /home/easybuild
+# disable output buffering in Python, so we see EasyBuild output as it progresses
+ENV PYTHONUNBUFFERED=TRUE
```

Comment:
ENV can be set at the top and will be present no matter the `USER` currently used.
```
FROM busybox

ENV TEST=1
USER nobody
RUN echo $TEST
```